src/controllers/document.js

```diff
@@ -136,10 +136,15 @@ class DocumentController extends BaseController {
     const request = {
       index,
       collection,
-      body: {ids},
-      action: 'mGet'
+      action: 'mGet',
+      includeTrash: options.includeTrash
     };
 
+    if (options.verb === 'POST') {
+      request.body = {ids};
+    } else {
+      request.ids = ids.toString();
```

Comment:
(Nitpicking) Prefer `Array.join` because it's more idiomatic. Also, the `else` is not indented regarding our coding standards